src/main/resources/scripts/simple god mode.sk

```diff
@@ -3,6 +3,6 @@
 #
 
 on damage:
-	victim is a player
-	victim has permission "skript.god"
-	cancel event

```

Comment:
Old code was perfectly valid. Actually I consider it better. Nesting if sections looks ugly and increases line width of script. In this case, it is also useless.

Reply:
Same as above, will revert